src/tests/admin/conferenceroomoutoforderplanning/IfOfOutOfOrderIsCreatedItsInformationIsDisplayedInOutOfOrderPage.java

```diff
@@ -0,0 +1,76 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomInfoPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.TimeManager;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC02: Verify Out of Order's information is displayed in Out Of Order planning if it is 
+ * for current time or for future time
+ * @author Yesica Acha
+ *
+ */
+public class IfOfOutOfOrderIsCreatedItsInformationIsDisplayedInOutOfOrderPage {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
```

Comment:
following java conventions, you should use access modifiers for your global variables